format/volume.go

```diff
@@ -95,7 +95,7 @@ func populateFieldFromBuffer(char rune, buffer []rune, volume *types.ServiceVolu
 			if isBindOption(option) {
 				setBindOption(volume, option)
 			}
-			// ignore unknown options
+			// ignore unknown options FIXME why not report an error here?
```

Comment:
I would vote to add a message here since the engine error message becomes cryptic to the user since these modes are set by compose so the engine can do its thing

Reply:
The engine has it's own parser for the bind string, which reject unknown options.
Docker Compose just must not send a bind string to engine when it can't translate into, but `String` can't return an error